### DISCUSSION TITLE
I'm not approved for student pack

### DISCUSSION THREAD

**Discussion by rav000** [community/community#19518] [GitHub Education]
I already rejected for 3 times, then I want to get it again but system said "We have received your application, and you should receive an email from us by June 25." and it's already in June 27. Here is a proof
![Screen Shot 2022-06-27 at 07 47 48](https://user-images.githubusercontent.com/97004117/175841335-7b81d3a4-a2a6-47ae-afb7-5ba2805eb14d.png)

**Comment by melvincwng** [comment]
@srv2003, you can raise a ticket and check with [Github Education Support](https://support.github.com/contact/education) on the status of your application.

Otherwise, please kindly wait a couple of working days more, for them to process your application.

Thanks.

If this answer has helped you, please 'Mark as answer' so that others know this issue has been addressed :)

**Comment by jonypeixoto** [comment]
Did you check if you used the email of the college or educational institution?

With educational email, you get accepted faster.
